### ISSUE TITLE
intermittent assertion failure in dump.c on travis

### ISSUE THREAD

**Issue by JeffBezanson** [JuliaLang/julia#12127]
I've seen this a couple times, e.g. https://travis-ci.org/JuliaLang/julia/jobs/70632635 :

```
julia: dump.c:1032: jl_deserialize_value: Assertion `bp' failed.
```

**Comment by tkelman** [comment]
So have I. Ref https://github.com/JuliaLang/julia/pull/12111#issuecomment-120665830

**Comment by vtjnash** [comment]
should be fixed by #8745
